### ISSUE TITLE
Nvim terminal color support

### ISSUE THREAD

**Issue by yzlnew** [sainnhe/gruvbox-material#24]
Terminal in Neovim support `g:terminal_color_N` to set colors. See in [here](https://neovim.io/doc/user/nvim_terminal_emulator.html#nvim-terminal-emulator-configuration).
A possible configuration for gruvbox-material medium.
```vim
if get(g:, 'colors_name', 'default') == 'gruvbox-material'
    let g:terminal_color_0  = '#282828'
    let g:terminal_color_1  = '#ea6962'
    let g:terminal_color_2  = '#a9b665'
    let g:terminal_color_3  = '#d8a657'
    let g:terminal_color_4  = '#7daea3'
    let g:terminal_color_5  = '#d3869b'
    let g:terminal_color_6  = '#89b482'
    let g:terminal_color_7  = '#d4be98'
    let g:terminal_color_8  = '#7c6f64'
    let g:terminal_color_9  = '#402120'
    let g:terminal_color_10 = '#34381b'
    let g:terminal_color_11 = '#e78a4e'
    let g:terminal_color_12 = '#0e363e'
    let g:terminal_color_13 = '#928374'
    let g:terminal_color_14 = '#374141'
    let g:terminal_color_15 = '#ddc7a1'
endif
```
Results on Windows10, Neovim with powershell
![image](https://user-images.githubusercontent.com/4904877/75435698-63a1d600-598e-11ea-9d61-86e46655aa06.png)

**Comment by sainnhe** [comment]
This is already supported, see [colors/gruvbox-material.vim](https://github.com/sainnhe/gruvbox-material/blob/89b1cdb33da73fdd09a57dc5ea156f5c2ccbb99f/colors/gruvbox-material.vim#L2149-L2164).

So doesn't that code work?

**Comment by yzlnew** [comment]
With default config,
![image](https://user-images.githubusercontent.com/4904877/75436842-28080b80-5990-11ea-8ff0-afc7b0f5c3d9.png)
Maybe the forground color is a little too gray? Not quite readable.
![image](https://user-images.githubusercontent.com/4904877/75437282-f3488400-5990-11ea-9cc7-77b1a0b44e74.png)

**Comment by sainnhe** [comment]
Should be fixed via fc2cf7a.

**Comment by yzlnew** [comment]
Thanks, works fine now.
